homeassistant/helpers/entity_platform.py

```diff
@@ -124,6 +125,7 @@ def __init__(
         self.scan_interval = scan_interval
         self.entity_namespace = entity_namespace
         self.config_entry: config_entries.ConfigEntry | None = None
+        # Store the entities for this specific platform only
```

Comment:
The verb "store" is unclear when creating an empty dict, can we rephrase it?
Also, it's very nice with the example below which points out "domain" means light, add an example here too since the way we use these terms is not consistent